### PR TITLE
response_time_ns handler metric

### PR DESCRIPTION
**Pull request by rodrigolck**
**Description**

Implement the response_time_ns for handler, so that it reports and we are able to see the time it takes to process the message.

### CODE REVIEW

File: pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs [hunk 1]
```diff
@@ -46,16 +51,30 @@ private static async Task HandleIncomingRpc(IntPtr cRpcPtr)
 
                 Marshal.FreeHGlobal(res.data);
                 Marshal.FreeHGlobal(resPtr);
+                if (success)
+                {
+                    MetricsReporters.ReportTimer(Metrics.Constants.Status.success.ToString(), route,
```
Comment (Henrod) [inline pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs]: Hardcoded to "" and "PIT-500"? What about the actual status code returned by the server?

File: pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs [hunk 2]
```diff
@@ -46,16 +51,30 @@ private static async Task HandleIncomingRpc(IntPtr cRpcPtr)
...
+                        "handler", "", sw);
```
Comment (Henrod) [inline pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs]: Isn't the correct type "rpc" instead of "handler"? Following the pitaya patterns

Comment (rodrigolck) [reply pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs]: If I saw it correctly, the ReportTimer containing "rpc" is already implemented, so I just needed to implement the other one

File: pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs (hunk 1 above)
Comment (rodrigolck) [reply pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs]: Copied from the other spot doing the same... Need to be able to access this Info

**Pull request merged**